metadata-io/src/main/java/com/linkedin/metadata/graph/elastic/ElasticSearchGraphService.java

```diff
@@ -326,6 +328,12 @@ public void configure() {
     }
   }
 
+  @Override
+  public String reindexAsync(String index, @Nullable QueryBuilder filterQuery, BatchWriteOperationsOptions options)
```

Comment:
At first glance, this doesn't seem right to me. The graph service index is not `timeseries` and would not be subject to timeseries-like truncation.